components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-profile/reporting-profile.component.ts

```diff
@@ -129,6 +129,10 @@ export class ReportingProfileComponent implements OnInit, OnDestroy {
   }
 
   getNodes(reportQuery: ReportQuery, params: any): Observable<Array<any>> {
+    // remove profile_id and control_id as filters if they are already applied
```

Comment:
I ended up opting to remove the profile_id and control_id filters just before loading in the new ones in the case that other filters were being applied, it wouldn't remove ALL filters, if we opted to add more.

Reply:
Initially I was clearing out ALL filters at the end of the function, which I still think may be the cleaner way to do it.